cookbook/import_export/clean-csv-file-during-product-import.rst

```diff
@@ -90,7 +90,7 @@ Then we declare this new array converter service in ``array_converters.yml``.
 
     You can notice here that we use the `Decorator Pattern`_ by injecting the default array converter in our own class.
 
-    The big advantage of this practise is to decouple your custom code from the PIM code, for instance, if in the future, an extra dependency is injected in the constructor of the default array converter, your code will not be impacted.
+    The big advantage of this practice is to decouple your custom code from the PIM code, for instance, if in the future, an extra dependency is injected in the constructor of the default array converter, your code will not be impacted.
```

Comment:
Sorry but not in this PR. I would not rewrite the entire stuff just fix these typos.